src/transport/http/binary_emitter.ts

```diff
@@ -0,0 +1,25 @@
+import axios, { AxiosRequestConfig } from "axios";
+
+import { CloudEvent, Version } from "../../event";
+import { Options } from "../emitter";
+import { Headers, headersFor } from "./headers";
+import { asData } from "../../event/validation";
+
+export async function emitBinary(event: CloudEvent, options: Options): Promise<object> {
+  if (event.specversion !== Version.V1 && event.specversion !== Version.V03) {
+    return Promise.reject(`Unknown spec version ${event.specversion}`)
+  } else {
+    return emit(event, options, headersFor(event));
```

Comment:
Seems like this `emit` method could be inlined. We only have a simple `if`/`else` here.